src/app/api/slack/userStatus/route.ts

```diff
@@ -0,0 +1,32 @@
+import { NextResponse } from "next/server";
+import axios from "axios";
+
+export async function GET(request: Request) {
+  const { searchParams } = new URL(request.url);
+  const userEmail = searchParams.get("userEmail");
+
+  if (!userEmail) {
+    return NextResponse.json(
+      { error: "User email is required" },
+      { status: 400 },
+    );
+  }
+  try {
+    const response = await axios.get(
+      `${process.env.SLACK_API_ROOT_PATH}/slack/user/status`,
+      {
+        params: { userEmail },
+        headers: {
+          "x-api-key": process.env.SLACK_API_TOKEN,
+        },
+      },
+    );
+
+    return NextResponse.json(response.data, { status: 200 });
+  } catch (error) {
```

Comment:
maybe we need to log the errors so we know what went wrong.
maybe later

Reply:
The next line is returning error but not printing in console.